### PR TITLE
Helm Chart for redis-commander

### PR DESCRIPTION
**Pull request by abdennour**
Congratulations!

### CODE REVIEW

**Comment by abdennour** [comment]
it solves #413

**Comment by sseide** [comment]
thanks - i will have a look at this

File: .helm-charts/redis-commander/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "redis-commander.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "redis-commander.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "redis-commander.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
```
Comment (sseide) [inline .helm-charts/redis-commander/templates/NOTES.txt:19]: Redis Commmander is running on port 8081 per default. Therefore these to lines are wrong and must be change to 8081 (insted of 8080 and 80)

**Comment by sseide** [comment]
done with commit 4c93819 as there were no changes within this PR as requested

**Pull request closed without merging**